packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -164,9 +164,10 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
         </TableBody>
       </MuiTable>
       <PaginationRow
+        page={pagination.page}
         offset={pagination.offset}
         first={pagination.first}
-        total={pagination.total}
+        total={pagination.total ?? 0}
```

Comment:
so close to being able to just spread 'em: `{...pagination}`

I think that others might request that all these props be destructured before use. I think fine like it is.